### DISCUSSION TITLE
Can `js-yaml` be called within Templater script?

### DISCUSSION THREAD

**Discussion by whytong** [SilentVoid13/Templater#1430] [Help] [answered]
I am not sure whether this or the Obsidian forum is the best place to ask the question but here it is:

I plan to create a Templater script to copy yaml header from another note and merge with the existing one in the active note. So I included the `const yaml = require('js-yaml');` in the script, but apparently, this does not work. I learn Obsidian uses `js-yaml` to clean up the YAML header, but it does not seem to be accessible. I wonder how can I access the js-yaml library in Templater scripts.

**Comment by Zachatoo** [comment, marked as answer]
There is not a way to import packages into Templater scripts.

You can however use Obsidian APIs in Templater scripts, such as [app.fileManager.processFrontMatter](https://docs.obsidian.md/Reference/TypeScript+API/FileManager/processFrontMatter).
